runnable.go

```diff
@@ -0,0 +1,54 @@
+package compute
+
+import (
+	"fmt"
+)
+
+type Runnable struct {
```

Comment:
We may want to steal the `Runnable` struct directly from Atmo? That would ensure that they're identical